### PR TITLE
style: openedx theme logo updates

### PR DESCRIPTION
**Pull request by mudassir-hafeez**
### What are the relevant tickets
[#3337](https://github.com/mitodl/hq/issues/3337)

### Description (What does it do?)
This PR updates logos in legacy edX theme for mitxonline.

### Screenshots (if appropriate):
Header and footer logo:
openedx header and footer logo

Studio header logo:
studio logo

Favicon:
openedx favicon

Emails (I couldn't test the actual emails but this is how it looks in the ace_templates):
logo in email template

### How can this be tested?

1. Enable Theme.
   - Follow the instructions in the [Readme](https://github.com/mitodl/mitxonline-theme).

2. Add Settings in private.py:

```
LMS_BASE_URL = "http://localhost:18000"
LOGO_URL_PNG_FOR_EMAIL = f"{LMS_BASE_URL}/static/mitxonline-theme/images/logo.png"
LOGO_URL = f"{LMS_BASE_URL}/static/mitxonline-theme/images/logo.svg"
LOGO_TRADEMARK_URL = f"{LMS_BASE_URL}/static/mitxonline-theme/images/mit-ol-logo.svg"
BULK_EMAIL_SEND_USING_EDX_ACE = True
```

3. Verify Logos:
    - Check the updated logos in the legacy theme and the MFEs.
    - For any MFE, you will need to set the LOGO_URL and LOGO_TRADEMARK_URL in the .env.development.

4. Verify Logo in Ace Emails:
    - I couldn't test the logo with this [format](https://github.com/mitodl/mitxonline-theme/pull/43/commits/04dd7f3d4866992e49c1f754ace1e0bd97bf1eb8#diff-8837e91672139956ef9117136e4c27f5167cd666374543107691b8d33e3ef935R65) locally due to some settings issue, but it seems to work when mapping the actual logo URL in the src.
    - Reviewers can test this format locally if possible, as this is being picked for xPRO. Otherwise, we will test this directly on RC.

### CODE REVIEW

File: cms/static/images/studio-logo.png
Comment (arslanashraf7) [inline cms/static/images/studio-logo.png:1]: @mbilalmughal Can we get a PNG version of this with bigger dimensions? We have seen blurry images issues in other themes for the PNG versions.

File: lms/static/images/logo.png
Comment (arslanashraf7) [inline lms/static/images/logo.png:1]: This should also be a larger e.g. 5x version of the image. This gives a blurry effect in the emails.

File: lms/templates/header/navbar-logo-header.html [hunk 1]
```diff
@@ -15,9 +15,9 @@
 
 <h1 class="header-logo">
     <div class="navbar-brand header">
-      <a href="https://web.mit.edu/" rel="noopener noreferrer" target="_blank" class="mit-link"></a>
-      <a href="${settings.MARKETING_SITE_BASE_URL}"" class="marketing-site-link"></a>
-      <img  class="logo" src="${branding_api.get_logo_url(is_secure)}" alt="${_("{platform_name} Home Page").format(platform_name=static.get_platform_name())}"/>
+      <a href="${settings.MARKETING_SITE_BASE_URL}">
```
Comment (arslanashraf7) [inline lms/templates/header/navbar-logo-header.html:18]: How flexible is this in the sense of adding a URL other than MARKETING_SITE_BASE_URL. Ideally, If possible we should send the user to https://openlearning.mit.edu/ from the trademark logo. I looked a bit but I don't see a suitable configuration for both MFEs and Legacy themes. Can you check in detail if we can set an explicit URL just for the trademark logo?

File: lms/templates/ace_common/edx_ace/common/base_body.html [hunk 2]
```diff
@@ -62,8 +62,8 @@
                     <tr>
                         <td width="70">
                             <a href="{% with_link_tracking site_configuration_values.HOMEPAGE_URL %}"><img
-                                    src="https://mitxonline.mit.edu/static/images/mitx-online-logo.png" width="auto"
-                                    height="40" alt="{% filter force_escape %}{% blocktrans %}Go to {{ platform_name }} Home Page{% endblocktrans %}{% endfilter %}"/></a>
+                                    src="{{ logo_url }}" width="125"
+                                    height="35" alt="{% filter force_escape %}{% blocktrans %}Go to {{ platform_name }} Home Page{% endblocktrans %}{% endfilter %}"/></a>
```
Comment (arslanashraf7) [inline lms/templates/ace_common/edx_ace/common/base_body.html:66]: 125x35 was a suitable lower size for xPRO logo based on It's original size. Is it suitable for MITx Online logo as well ? The dimensions I see from the [Figma link](https://www.figma.com/design/GFl0GZREDaWmdRytsNiET2/MIT-Logo-Header-%2F-Footer?node-id=1-2&t=FaHM0xe4CHQVWD6X-0) are 121.55x33.

File: lms/templates/footer.html [hunk 3]
```diff
@@ -103,9 +103,9 @@
 
         <div class="wrapper-logo">
             <div class="navbar-brand">
-                <a href="https://web.mit.edu/" rel="noopener noreferrer" target="_blank" class="mit-link"></a>
-                <a href="${settings.MARKETING_SITE_BASE_URL}"" class="marketing-site-link"></a>
-                <img  class="logo" src="${footer['logo_image']}" alt="${_("{platform_name} Home Page").format(platform_name=static.get_platform_name())}"/>
+                <a href="${settings.MARKETING_SITE_BASE_URL}">
+                  <img class="logo mitxonline-ol-logo" src="${footer['logo_image']}" alt="${_("{platform_name} Home Page").format(platform_name=static.get_platform_name())}"/>
```
Comment (arslanashraf7) [inline lms/templates/footer.html:107]: I think the CSS size isn't getting picked here, I'm seeing an image of 164x42.
image

**Review by arslanashraf7 — CHANGES_REQUESTED** [review]

File: cms/static/images/studio-logo.png
Comment (mbilalmughal) [reply cms/static/images/studio-logo.png:1]: @arslanashraf7 Sure, Here it is 
mitx-online

Comment (mbilalmughal) [reply cms/static/images/studio-logo.png:1]: Here is the transparent one in case you need it without white background
mitx-online

Comment (mbilalmughal) [reply cms/static/images/studio-logo.png:1]: @mudassir-hafeez, as requested, here is 512 dimension logos, (white background & transparent)
mitx-online-logo-512
mitx-online-logo-512-transperent

Comment (mudassir-hafeez) [reply cms/static/images/studio-logo.png:1]: Thanks @mbilalmughal, only concern was to use what @arslanashraf7 mentioned as we have seen blurry images issues in other themes for the PNG versions as mentioned above.

**Comment by mudassir-hafeez** [comment]
@arslanashraf7 I've made most of the changes but could not test lms side or mfes as there were some issues with local setup. Could you please test the changes at at your end? however, I could check/test studio log at my side.

File: lms/templates/header/navbar-logo-header.html (hunk 1 above)
Comment (mudassir-hafeez) [reply lms/templates/header/navbar-logo-header.html:18]: I've looked into it, I also didn't get any suitable configuration. I think, this is same we are using for xpro. So we can use it as it is, and can pick in the future as it is required a thorough investigation.

**Review by arslanashraf7 — CHANGES_REQUESTED** [review]
In addition to the inline comment(s). I think the logo on the homepage and the legacy courseware pages e.g. Progress page looks way too big. Can you match that with the size of the logo on the marketing app https://rc.mitxonline.mit.edu/?

**Comment by mudassir-hafeez** [comment]
> In addition to the inline comment(s). I think the logo on the homepage and the legacy courseware pages e.g. Progress page looks way too big. Can you match that with the size of the logo on the marketing app https://rc.mitxonline.mit.edu/?

This is how it looks like now with 121x33 what we have on https://rc.mitxonline.mit.edu/:

progress page:
progress page

dashboard:
homepage

File: lms/templates/ace_common/edx_ace/common/base_body.html (hunk 2 above)
Comment (mudassir-hafeez) [reply lms/templates/ace_common/edx_ace/common/base_body.html]: It seems to me 124.3x33px as per [figma](https://www.figma.com/design/GFl0GZREDaWmdRytsNiET2/MIT-Logo-Header-%2F-Footer?node-id=1-2&t=FaHM0xe4CHQVWD6X-0). So I set this size for now. Will update if this needs adjustment.

**Review by arslanashraf7 — APPROVED** [review]
Looks good 👍  Could you please get the latest screenshots approved by @pdpinch before you merge?

**Post Merge**
There are a couple of things you will need post merge:

1. PR in https://github.com/mitodl/ol-infrastructure. I did merge some supporting changes for MITx Online in https://github.com/mitodl/ol-infrastructure/pull/2308 but you will need to check if something is missing
2. The course email templates are saved in Django Admin, You will need to change the URLs in those in order to update the logo there as well.

**Comment by mudassir-hafeez** [comment]
@pdpinch, could you please review the latest screenshots attached in the PR description for approval before we proceed with merging? Thank you!

**Comment by pdpinch** [comment]
Screenshots LGTM. You'll need to coordinate with devops to deploy this to QA/RC and prod.

**Pull request merged**